scrapers/RealityKingsOL.yml

```diff
@@ -30,7 +30,12 @@ xPathScrapers:
           - parseDate: January 2, 2006
       Details: //span/div[text()="Description:"]/following-sibling::text()
       Tags:
-        Name: //span[div[text()='Categories:']]/a/text()[1]
+        Name:
+          selector: //span[div[text()='Categories:']]/a/text()[1]
+          postProcess:
+            - replace:
+                - regex: \,
```

Comment:
Tags in most sites don't seem to have commas as part of them so removing all commas might be ok
Just to be safe can't we only replace leading or trailing commas though?
Something like 
```yaml
                - regex: \,$|^\,
```
?

Reply:
Good point. Might get thrown if there's whitespace on the ends of the tags - is whitespace ever an issue on tag ends?

How about
`
                - regex: \,\s*$|^\s*\,`

Or even

`                - regex: \s*\,?\s*$|^\s*\,?\s*`

That's spaces, a possible comma and then more spaces - so good against " , tag , " or "  tag  " but doesn't break "Tit, Large"

(Am I overthinking this?)

Reply:
i think the first you mention (`- regex: \,\s*$|^\s*\,`) is enough. Even my suggestion was just to be safe...